docker-compose.yml

```diff
@@ -194,10 +194,38 @@ services:
       databaseSchemaName: timecard
       <<: *db-env
 
+  person-schema-creation:
+    image: quay.io/ukhomeofficedigital/postgres:latest
+    environment:
+      <<: *db-env
+    command:
+      - "sh"
+      - "-c"
+      - "export PGPASSWORD=$DATABASE_PASSWORD; psql postgresql://postgres:5432/callisto -U postgres -c 'CREATE SCHEMA IF NOT EXISTS person;'"
```

Comment:
For this to be made into a reusable YAML anchor, there has to be a way to pass on the DB schema name (e.g. person, accruals, etc.) as a parameter to the YAML anchor so it can be injected in the create schema statement.
I couldn't find a way to achieve this in YAML/Docker. Are you aware of a simple way this can be achieved?

Reply:
Good shout. Actioned